docs/dev-proc/versioning.md

```diff
@@ -14,66 +14,94 @@ in menu on the left side (Supported Hardware->Hardware Model->Releases).
+(Supported Hardware->Hardware Model->Releases).
+
+## PC Engines releases and its exceptional versioning scheme
+
+Dasharo (coreboot+SeaBIOS) for PC Engines does not adhere to the typical
+versioning scheme used by Dasharo, which is semantic versioning. This is due to
+a couple of reasons:
+
+- We want to convey that this series of releases is a direct continuation of
+  past efforts sponsored by PC Engines and published
+  [here](https://pcengines.github.io/). By sticking to that versioning scheme, we
+  do not deviate from the agreed pattern and what users are accustomed to.
+- We have utilized the same infrastructure, and maintaining versioning
+  according to the pattern used by us since 2017 has helped us minimize changes.
+
+The general rule for versioning pattern is as follows: `<coreboot_rel_ver>{.00.}<dasharo_rel_num>`
+
+- `<coreboot_rel_ver>` - changed in February 2024, from x.y.z to YY.MM{.FF},
+  where `YY` represents the year, `MM` represents the month, and optional `{.FF}`
+  represents the patch number if any hotfix for the given release is created.
+- `{.00.}` - if no hotfix exists for the given coreboot release, we add `.00.`
+  to reserve space for a potential hotfix on that version.
+- `<dasharo_rel_num>` - indicates the number of versions released by the
+  Dasharo Team on top of the given coreboot release, starting from `01`, where
+  version `00` means no changes were applied on top of the coreboot release.
 
 ## Signing keys
 
 In Dasharo we use following rules for keys:
 
-* GPG RSA 4096 for signing and authentication and subkey for encryption
-* There few types of naming conventions, which define `Real Name` field and
+- GPG RSA 4096 for signing and authentication and subkey for encryption
+- There few types of naming conventions, which define `Real Name` field and
   chain of trust schemes:
-    - Software:
-        + Real Name: `<name> open-source software release <version> signing key`
-        + Signing key: `3mdeb Open Source Software Master Key <contact@3mdeb.com>`
-    - Firmware:
-        + Real Name: `<name> open-source firmware release <version> signing key`
-        + Signing key: `3mdeb Dasharo Master Key`
-    - PC Engines (firmware exception):
-        + Real Name: `PC Engines open-source firmware release <version> signing key`
-        + Signing key: `3mdeb Open Source Firmware Master Key <contact@3mdeb.com>`
-    - Dasharo firmware produced by 3mdeb:
-        + Real Name: `Dasharo release <version> compatible with <name> signing key`
-        + Signing key: `3mdeb Dasharo Master Key`
-    - For Dasharo firmware produced by 3mdeb on customer's behalf:
-        + Real Name: `Dasharo open-source firmware <version> for <name> signing key`
-        + Signing key: `3mdeb Dasharo Master Key`
+    + Software:
+        * Real Name: `<name> open-source software release <version> signing key`
+        * Signing key: `3mdeb Open Source Software Master Key <contact@3mdeb.com>`
+    + Firmware:
```

Comment:
We can drop it. Only two keys are generated in that way: Tuxedo 1.x and PC Engines 4.19. Both are expired at this point.

Reply:
https://github.com/Dasharo/docs/pull/841/commits/937e5c07872200f133573c328262d771b179d8d0#diff-2fbf769fce1a51e7078b34a3c1fcfcb3c5b376ae925ba1c0d5954aaecb0ed34cL83-L85

Reply:
Actually three at least: `Novacustom Open Source Firmware Release 1.x Signing Key` I mentioned earlier.

I think we should keep the `Firmware` type of key and mention it is deprecated, but we provide the format, because some active keys still use this format.

Reply:
> Actually three at least: Novacustom Open Source Firmware Release 1.x Signing Key I mentioned earlier

Ok, the name is different. That is why my query did not catched it:

```sh
uid                 [ expired] PC Engines Open Source Firmware Release 4.9 Signing Key
uid                 [ expired] PC Engines Open Source Firmware Release 4.13 Signing Key
uid                 [ expired] PC Engines Open Source Firmware Release 4.14 Signing Key
uid                 [ expired] Novacustom Open Source Firmware Release 0.x Signing Key
uid                 [ expired] PC Engines Open Source Firmware Release 4.15 Signing Key
uid                 [ expired] Novacustom Open Source Firmware Release 1.x Signing Key
uid                 [ expired] Tuxedo open-source firmware release 1.x signing key
uid                 [ expired] PC Engines open-source firmware release 4.19 signing key
```

> I think we should keep the Firmware type of key and mention it is deprecated, but we provide the format, because some active keys still use this format.

Ok. Please check a9584c8e6f90

Reply:
Yes, we also changed from capital to small letters. Looks good.